addons/mail/static/src/models/message_notification_view.js

```diff
@@ -0,0 +1,42 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear, replace } from '@mail/model/model_field_command';
+
+const { markup } = owl;
+
+registerModel({
+    name: 'MessageNotificationView',
+    identifyingFields: ['messageListViewItemOwner'],
```

Comment:
Rebase master and adapt with change with identifying fields.
Here it's just putting `identifying: true` on `messageListViewItemOwner`

Reply:
Yes, I wasn't fully rebased :)